src/_pytest/faulthandler.py

```diff
@@ -39,7 +45,11 @@ def pytest_unconfigure(config: Config) -> None:
         del config.stash[fault_handler_stderr_fd_key]
     if config.stash.get(fault_handler_originally_enabled_key, False):
         # Re-enable the faulthandler if it was originally enabled.
-        faulthandler.enable(file=get_stderr_fileno())
+        if fault_handler_original_stderr_fd_key in config.stash:
+            faulthandler.enable(config.stash[fault_handler_original_stderr_fd_key])
+            del config.stash[fault_handler_original_stderr_fd_key]
+        else:
+            faulthandler.enable(file=get_stderr_fileno())
```

Comment:
Wondering, any reason why we check `if fault_handler_original_stderr_fd_key in config.stash`? Because we do something similar in line 43?

Given we unconditionally set `fault_handler_original_stderr_fd_key`, seems safe to always assume it is present? This would also solve the missing coverage check.

```suggestion
        faulthandler.enable(config.stash[fault_handler_original_stderr_fd_key])
        del config.stash[fault_handler_original_stderr_fd_key]
```